composer.json

```diff
@@ -19,9 +19,9 @@
     "doctrine/doctrine-bundle": "^1.10|^2.0",
     "doctrine/orm": "^2.7",
     "ramsey/uuid-doctrine": "^1.5",
-    "symfony/workflow": "^4.4|^5.2",
-    "symfony/validator": "^4.4|^5.2",
-    "symfony/framework-bundle": "^4.4|^5.2"
+    "symfony/workflow": "^4.4|^5.4|^6.0",
+    "symfony/validator": "^4.4|^5.4|^6.0",
+    "symfony/framework-bundle": "^4.4|^5.4|^6.0"
```

Comment:
waarom doen we hier dan 6.0? en niet 6?

Reply:
Symfony 6 ondersteunt php 8 en symfony 6.1 php 8.1. Het voelt beter om direct 8.1 te ondersteunen dan nu op 8.0 te blijven

Reply:
Dit kan natuurlijk ook, ik heb denk ik uit automatisme gewoon 6.0 neergezet. 
Zal het aanpassen naar ^6

Reply:
@dragondungeon1 Naar mijn mening is je automatisme goed geweest;
het is gebruikelijk om de versies als ^6.0 te noteren; zie bijvoorbeeld hoe de grote jongens het doen;
Symfony: https://github.com/symfony/symfony/blob/6.2/composer.json
ApiPlatform: https://github.com/api-platform/core/blob/main/composer.json
EasyAdmin: https://github.com/EasyCorp/EasyAdminBundle/blob/4.x/composer.json

Graag terugdraaien naar ^6.0

@rvmourik ter info

Reply:
Het was ook een vraag, geen requested change :-)

Reply:
oh oke, thank you zal het weer terug draaien :)